statsmodels/tools/eval_measures.py

```diff
@@ -67,6 +68,30 @@ def rmse(x1, x2, axis=0):
     x2 = np.asanyarray(x2)
     return np.sqrt(mse(x1, x2, axis=axis))
 
+def rmspe(y, y_hat, axis=0, zeros=np.nan):
+    """
+    root mean squared percentange error
+    
+    Parameters
+    ----------
+    y, y_hat : array_like
```

Comment:
One parameter per entry, so you will need two lines here. Multiple entries do not render correctly in sphinx.

Reply:
Again, here I'm just following the style of the rest of the script.

Reply:
New content needs to conform to current standards. There is a lot of old code that was developed with fewer standards in place and fewer tools to verify them.

Reply:
Done now